neutrino-custom/base.js

```diff
@@ -36,8 +36,7 @@ module.exports = neutrino => {
             'angular', 'angular-cookies', 'angular-local-storage', 'angular-resource',
             'angular-route', 'angular-sanitize', 'angular-toarrayfilter', 'angular-ui-bootstrap',
             'angular-ui-router', 'bootstrap/dist/js/bootstrap', 'hawk', 'jquery', 'jquery.scrollto',
-            'js-yaml', 'mousetrap', 'react', 'react-dom', 'taskcluster-client', 'numeral'
-        ];
+            'js-yaml', 'mousetrap', 'react', 'react-dom', 'taskcluster-client', 'numeral', 'metrics-graphics'        ];
```

Comment:
I think the `];` part should stay where it was.

Probably need to update the eslint rules here.